po/th.po

```diff
@@ -0,0 +1,573 @@
+msgid "Pause"
+msgstr "หยุดชั่วคราว"
+
+#: src/plattenalbum.py:2349
+#, python-brace-format
+msgid "{bitrate} kb/s"
+msgstr "{bitrate} kb/s"
+
+#: src/plattenalbum.py:2374 data/ShortcutsWindow.ui:107
+msgid "Previous"
+msgstr "ก่อนหน้า"
+
+#: src/plattenalbum.py:2375 data/ShortcutsWindow.ui:101
+msgid "Next"
+msgstr "ถัดไป"
+
+#: src/plattenalbum.py:2387
+msgid "Progress bar"
+msgstr "แถบความคืบหน้า"
+
+#: src/plattenalbum.py:2501
+msgid "Volume control"
+msgstr "ควบคุมความดัง"
+
+#: src/plattenalbum.py:2521
+msgid "Player Menu"
+msgstr "เมนูการเล่นเพลง"
+
+#: src/plattenalbum.py:2531
+msgid "_Repeat Mode"
+msgstr "เล่นซ้ำ"
+
+#: src/plattenalbum.py:2532
+msgid "R_andom Mode"
+msgstr "สุ่ม"
+
+#: src/plattenalbum.py:2533
+msgid "_Single Mode"
+msgstr "เล่นเพลงเดี่ยว"
+
+#: src/plattenalbum.py:2534
+msgid "_Pause After Song"
+msgstr "หยุดหลังเพลงจบ"
+
+#: src/plattenalbum.py:2535
+msgid "_Consume Mode"
+msgstr "Comsume"
+
+#: src/plattenalbum.py:2538
+msgid "_Lyrics"
+msgstr "เนื้อเพลง"
+
+#: src/plattenalbum.py:2831
+msgid "Database is being updated"
+msgstr "ฐานข้อมูลกำลังถูกอัพเดต"
+
+#: src/plattenalbum.py:2832
+msgid "Database updated"
+msgstr "ฐานข้อมูลถูกอัพเดตเรียบร้อย"
+
+#. status page
+#: src/plattenalbum.py:2876
+msgid "Connect to Your Music"
+msgstr "เชื่อมต่อกับเพลงของคุณ"
+
+#: src/plattenalbum.py:2877
+msgid ""
+"To use Plattenalbum, an instance of the Music Player Daemon needs to be set "
+"up and running on this device or another one on the network"
+msgstr ""
+"ในการใช้งาน Plattenalbum นั้น Music Player Daemon จำเป็นต้องถูกตั้งค่าและ"
+"เปิดการใช้งานบนอุปกรณ์นี้ หรือบนเน็ตเวิร์ก"
+
+#: src/plattenalbum.py:2879
+msgid "_Set up Instance"
+msgstr "เซ็ตอัพเริ่มต้น"
+
+#: src/plattenalbum.py:2881
+msgid "Connect _Locally"
+msgstr "เชื่อมต่อแบบ Local"
+
+#: src/plattenalbum.py:2883
+msgid "Connect _Remotely"
+msgstr "เชื่อมต่อแบบ Remote"
+
+#: src/plattenalbum.py:2989
+msgid "Next Title is Playing"
+msgstr "เพลงต่อไปกำลังเล่น"
+
+#: src/plattenalbum.py:2991
+#, python-brace-format
+msgid "Now playing “{title}” by “{artist}”"
+msgstr "ขณะนี้กำลังเล่น “{title}” โดย “{artist}”"
+
+#: src/plattenalbum.py:2993
+#, python-brace-format
+msgid "Now playing “{title}”"
+msgstr "ขณะนี้กำลังเล่น “{title}”"
+
+#: src/plattenalbum.py:3001
+msgid "Playback Finished"
+msgstr "การเล่นเพลงเสร็จแล้ว"
+
+#: src/plattenalbum.py:3002
+msgid "The playlist is over"
+msgstr "เพลย์ลิสต์จบแล้ว"
+
+#: src/plattenalbum.py:3031
+msgid "Cleared A‐B loop"
+msgstr "ลบลูป A-B แล้ว"
+
+#: src/plattenalbum.py:3034
+#, python-brace-format
+msgid "Started A‐B loop at {start}"
+msgstr "เริ่ม A-B loop ที่ {start}"
+
+#: src/plattenalbum.py:3036
+#, python-brace-format
+msgid "Activated A‐B loop from {start} to {end}"
+msgstr "เริ่ม A-B loop จาก {start} ถึง {end}"
+
+#: src/plattenalbum.py:3053
+msgid "Debug mode"
+msgstr "โหมด Debug"
+
+#: src/plattenalbum.py:3110
+msgid "translator-credits"
+msgstr "Supakorn Huemwang <shuemwang.han@gmail.com>\n"
```

Comment:
```suggestion
msgstr "Supakorn Huemwang <shuemwang.han@gmail.com>"
```

This additional newline seems to be a problem.

Reply:
I see, I will try committing new changes without the newline.